containers/fhir-converter/app/main.py

```diff
@@ -206,6 +207,23 @@ def add_rr_data_to_eicr(rr, ecr):
         # Append the ecr section into the eCR - puts it at the end
         ecr.append(ecr_section)
 
+    # If eICR >=R3, remove (optional) RR section that came from eICR
+    # This is duplicate/incomplete info from RR
+    ecr_version = ecr.xpath('//*[@root="2.16.840.1.113883.10.20.15.2"]/@extension')
+    if (len(ecr_version) > 0) and (
+        ecr_version[0] == "2021-01-01" or ecr_version[0] == "2022-05-01"
+    ):
+        namespaces = {"hl7": "urn:hl7-org:v3"}
+        rr_from_eicr = ecr.xpath(
+            '//hl7:component[hl7:section/hl7:templateId[@root="2.16.840.1.113883.10.20.15.2.2.5" and @extension="2021-01-01"]]',
+            namespaces=namespaces,
```

Comment:
should we remove this rr info before adding in the sepearate rr? I don't know if this is founded, but it feels slightly safer to remove things before we add things just to be extra sure we don't remove info from the rr